src/Exercism.Analyzers.CSharp/Analyzers/WeighingMachine/WeighingMachineSolution.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Exercism.Analyzers.CSharp.Syntax;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Exercism.Analyzers.CSharp.Analyzers.WeighingMachine
+{
+    internal class WeighingMachineSolution : Solution
+    {
+        public const string WeighingMachineClassName = "WeighingMachine";
+        public const string PrecisionPropertyName = "Precision";
+        public const string WeightPropertyName = "Weight";
+        public const string TareAdjustmentPropertyName = "TareAdjustment";
+        public const string DisplayWeightPropertyName = "DisplayWeight";
+        public const string WeightFieldName = "weight";
```

Comment:
Is this checking the backing field of the property? This could be problematic, as we usually don't comment on the naming of things (unless it breaks the tests) and I'm wondering whether this would still work then?

Reply:
The idea was to check that backing field is private. I can try thinking of another approach for this.

Reply:
Maybe you could try to get the name of the backing field from the property definition?

Reply:
Yes, I was wondering about this

Reply:
It will be slightly finicky, but it is doable I think